.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -108,24 +106,32 @@ jobs:
           TARGET_BRANCH: ${{ inputs.target-branch }}
           RELEASE: ${{ inputs.stage == 'release' && 'yes' || 'no' }}
 
+      - name: Parse OCI image name and digest
+        id: parse-oci-segments
+        if: ${{ env.PERFORM_ATTESTATION == 'yes' }}
+        run: |
+          oci_image="$(<"build/${{ matrix.asset }}-oci-image")"
```

Comment:
Sorry for the dumb question - do we need `$(readlink -f build)` here, or is the point that the pipe into can follow symlinks just fine?

Reply:
yes, I think the comment indicates resolving the build symlink is required for archiving. I don't think we need this for this purpose